doc/api/tls.md

```diff
@@ -1431,10 +1431,11 @@ changes:
     PEM allows the option of private keys being encrypted. Encrypted keys will
     be decrypted with `options.passphrase`. Multiple keys using different
     algorithms can be provided either as an array of unencrypted key strings or
-    buffers, or an array of objects in the form `{pem: <string|buffer>[,
-    passphrase: <string>]}`. The object form can only occur in an array.
-    `object.passphrase` is optional. Encrypted keys will be decrypted with
-    `object.passphrase` if provided, or `options.passphrase` if it is not.
+    buffers, or an array of objects in the form
+    `{pem: <string|buffer>[, passphrase: <string>]}`. The object form can only
```

Comment:
Breaking across lines for code element renders OK in GitHub and the rendered HTML, but doesn't in IDEs and is flagged by Markdownlint